shared/k8s/debug.go

```diff
@@ -24,3 +40,17 @@ func FetchValidatorPrivateKeys(clientset *kubernetes.Clientset) (map[string]stri
 	}
 	return validatorKeysMap, nil
 }
+
+func getNamespace() (string, error) {
+	nsFile := "/var/run/secrets/kubernetes.io/serviceaccount/namespace"
+
+	if _, err := os.Stat(nsFile); err == nil {
```

Comment:
NIT: Easier codepath is to do a "err !=nil; return "default" so you're indenting less code.